src/storage/mod.rs

```diff
@@ -6909,4 +6913,70 @@ mod tests {
             .unwrap();
         rx.recv().unwrap();
     }
+
+    #[test]
+    fn test_txn_cmd_with_ttl_enabled() {
+        let storage = TestStorageBuilder::new(DummyLockManager {}, true)
+            .build()
+            .unwrap();
+        let (tx, rx) = channel();
+
+        // TTL commands can pass the check.
+        storage
+            .raw_batch_put(
+                Context::default(),
```

Comment:
The `for_cas` field should be enabled if you want it to be a scheduler command.

Reply:
I missed  the `_atomic` suffix...